storybook/storybook-docs/src/typography.stories.mdx

```diff
@@ -80,8 +80,22 @@ Voor verdere informatie: [Wrapping and breaking text op MDN](https://developer.m
 
 ## Andere talen
 
-[Geef programmatisch aan in welke taal tekst in een afwijkende taal is](https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html).
-Dit is onder andere nodig zodat schermlezers deze tekst in de juiste taal uitspreken.
+Geef programmatisch aan in welke taal tekst in een afwijkende taal is. Dit is onder andere nodig zodat schermlezers deze tekst in de juiste taal uitspreken.
+
+```html
+<html lang="en-GB">
+  <head>
+    ...
+  </head>
+  <body>
+    <p>As the Latin saying goes — <q lang="la">in vino veritas</q></p>
+  </body>
+</html>
+```
```

Comment:
Let’s use actual content where we can – this may be a good example?

```suggestion
```html
<ul>
  <li><a href="https://mijn.amsterdam.nl/">Mijn Amsterdam</a></li>
  <li><a href="https://www.amsterdam.nl/en/" lang="en">English site</a></li>
</ul>
```
```

Reply:
I'd like to keep the `<html lang=XX>` part though, that shows how you set the generic language vs diverging language